docs/standalone/no_attestation.sh

```diff
@@ -16,9 +16,16 @@
 set -e
 
 # Create a public and private key pair.
-gpg --quick-generate-key --yes attestor@example.com
-gpg --armor --export attestor@example.com > gpg.pub
-gpg --armor --export-secret-keys attestor@example.com > gpg.priv
+GPG_OUTPUT="$(gpg --quick-generate-key --yes attestor@example.com)"
+
+# Save its fingerprint.
+KEY_FINGER_PRINT="$(echo $GPG_OUTPUT | sed -n 's/.*\([A-Z0-9]\{40\}\).*/\1/p')"
+if [ ${#KEY_FINGER_PRINT} -ne 40 ]; then echo "Error: key fingerprnt is not 40 characters." ; exit
+else echo "Generated key fingerprint is $KEY_FINGER_PRINT."
+fi
+
+gpg --armor --export $KEY_FINGER_PRINT > gpg.pub
```

Comment:
nit: let's keep this as one word, FINGERPRINT

Reply:
ack

Reply:
I think this is the only one left, maybe you have an unexported commit?

Reply:
hmm, It should have been fixed in this [commit](https://github.com/grafeas/kritis/pull/388/commits/983356e6b8ed20c5a40ac7f68c80418996b64a3e). Could you check the overall file changes? I should probably have hit `resolve` though.